apps/condo/domains/organization/schema/ResetOrganizationService.js

```diff
@@ -113,6 +114,38 @@ const ResetOrganizationService = new GQLCustomSchema('ResetOrganizationService',
                     await OrganizationLink.softDelete(context, organizationLink.id, DV_SENDER)
                 }
 
+                const meters = await find('Meter', {
+                    deletedAt: null,
+                    organization: { id: organizationId },
+                })
+                for (let meter of meters) {
+                    await Meter.softDelete(context, meter.id, DV_SENDER)
+                }
+
+                const meterResourceOwners = await find('MeterResourceOwner', {
+                    deletedAt: null,
+                    organization: { id: organizationId },
+                })
+                for (let meterResourceOwner of meterResourceOwners) {
+                    await MeterResourceOwner.softDelete(context, meterResourceOwner.id, DV_SENDER)
+                }
+
+                const propertyMeters = await find('PropertyMeter', {
+                    deletedAt: null,
+                    organization: { id: organizationId },
+                })
+                for (let propertyMeter of propertyMeters) {
+                    await PropertyMeter.softDelete(context, propertyMeter.id, DV_SENDER)
+                }
+
+                const meterReportingPeriods = await find('MeterReportingPeriod', {
```

Comment:
we can leave it. but you need to check that we will not send a notifications for that meters

Reply:
is done